doc/api/plural/plural-generator.md

```diff
@@ -13,9 +13,20 @@ to return the plural group.
 
 A Number for which to return the plural group.
 
+**options** Optional
+
+A JSON object including none or any of the following options.
+
+>> **type** Optional
+>>
+>> String `cardinal` (default), or `ordinal`.
+
```

Comment:
Please, could you move *options* above *value*? Also, replace `>>` with `>` only.